scripts/performance/requirements.txt

```diff
@@ -0,0 +1 @@
+psutil==6.1.1
```

Comment:
I was wondering if we want a lockfile, but when testing locally it looks like `psutil` would be the only thing included so I think that's okay.